horovod/spark/common/store.py

```diff
@@ -164,6 +165,15 @@ def read(self, path):
         with self.get_filesystem().open(self.get_localized_path(path), 'rb') as f:
             return f.read()
 
+    def read_serialized_keras_model(self, ckpt_path, model=None):
+        """
+        This function will be overridden by DBFSLocalStore.
+        """
+        from horovod.runner.common.util import codec
+
+        model_bytes = self.read(ckpt_path)
+        return codec.dumps_base64(model_bytes)
```

Comment:
beyond this PR: base64 is not very efficient in encoding bytes. Why not transferring the model bytes directly? cc: @tgaddair

Reply:
The original intent was to be able to store as part of a JSON file that encoded Estimator params, so we would represent the model as a base64 encoded string.  I believe this was changed, however, so may not be needed anymore.

@abditag2 can confirm.

Reply:
Correct. When we serialize the estimator, the model is stored as a b64 encoded string in Json along with the rest of the parameters of the SparkMLEstimator.

Reply:
Thanks, so we may not need it here then since we're reading from disk, but this can be addressed in a follow-up PR.

Reply:
Yes.